src/main/resources/openapi/dailylist.yaml

```diff
@@ -97,7 +97,7 @@ paths:
         - in: header
           name: json_document
           schema:
-            $ref: '#/components/schemas/PostDailyListJson'
+            type: string
```

Comment:
The use of a string is consistent with our use of the HTTP header containing the xml (see xml_document header). 

Additionally this change was primarily made as I was seeing feign encode decode issues, whereby json was not being transmitted in the HTTP header correctly.

I suspect the reason for this is that we cannott expect feign to encode and decode http headers automatically for us (I have tested this and it does not attempt to call onto the jackson ObjectMapper in such cases). Feign only encodes and decodes around the HTTP body:)). This makes sense to me as headers don't have content type meta data.

If we did want to retain the use of the specific http data typewe would have to write custom decode encode logic around the HTTP header